### PR TITLE
there was a merge issue

### PR DESCRIPTION
**Pull request by RamboV**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by RamboV** [comment]
@sarah-yo any update on the request?

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS
+author: Paul van Gool, HYAS Infosec
+
+This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with current WHOIS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the [connector documentation](https://docs.microsoft.com/en-us/connectors/hyasinsight/) or visit [HYAS Insight](https://www.hyas.com/contact) to request a trial key.
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/readme.md:4]: Please remove the "en-us" from this link

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Historic-WHOIS/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Enrich-Sentinel-Incident-HYAS-Insight-Domain-Historic-WHOIS
+author: Paul van Gool, HYAS Infosec
+
+This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with historic WHOIS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the [connector documentation](https://docs.microsoft.com/en-us/connectors/hyasinsight/) or visit [HYAS Insight](https://www.hyas.com/contact) to request a trial key.
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Historic-WHOIS/readme.md:4]: Please remove the "en-us" from this link

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Passive-DNS/readme.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Enrich-Sentinel-Incident-HYAS-Insight-Domain-Passive-DNS
+author: Paul van Gool, HYAS Infosec
+
+This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with recent passive DNS records. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the [connector documentation](https://docs.microsoft.com/en-us/connectors/hyasinsight/) or visit [HYAS Insight](https://www.hyas.com/contact) to request a trial key.
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Passive-DNS/readme.md:4]: Please remove the "en-us" from this link

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Email-Dynamic-DNS/readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Enrich-Sentinel-Incident-HYAS-Insight-Email-Dynamic-DNS
+author: Paul van Gool, HYAS Infosec
+
+This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with Dynamic DNS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the [connector documentation](https://docs.microsoft.com/en-us/connectors/hyasinsight/) or visit [HYAS Insight](https://www.hyas.com/contact) to request a trial key.
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Email-Dynamic-DNS/readme.md:4]: Please remove the "en-us" from this link

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Email-Historic-WHOIS/readme.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Enrich-Sentinel-Incident-HYAS-Insight-Email-Historic-WHOIS
+author: Paul van Gool, HYAS Infosec
+
+This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with historic WHOIS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the [connector documentation](https://docs.microsoft.com/en-us/connectors/hyasinsight/) or visit [HYAS Insight](https://www.hyas.com/contact) to request a trial key.
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Email-Historic-WHOIS/readme.md:4]: Please remove the "en-us" from this link

**Review by sarah-yo — CHANGES_REQUESTED** [review]
Please remove the "en-us" from any Microsoft document links within this PR as it will make the document validation fail. Thanks!

**Review by sarah-yo — CHANGES_REQUESTED** [review]
There are still locale links in the files - they all need to be removed for the documents link validation to pass.

**Comment by RamboV** [comment]
> There are still locale links in the files - they all need to be removed for the documents link validation to pass.

HI , I am not getting where the actual URL validation is failing, I completed removed the HYAS insight connector link for one commit even than its failing, can you tell us where exactly we need to change and what

File: Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,368 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+	"metadata": {
+        "comments": "This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with current WHOIS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the https://docs.microsoft.com/en-us/connectors/hyasinsight/ or visit https://www.hyas.com/contact to request a trial key.",
```
Comment (sarah-yo) [inline Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/azuredeploy.json:5]: Here is an example where the locale part of the URL is still present

Comment (RamboV) [reply Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/azuredeploy.json]: Thank you @sarah-yo , I have updated the links accordingly...

**Pull request merged**